rqd/rqd.example.conf

```diff
@@ -0,0 +1,22 @@
+# An Example rqd config file.
+# It can be placed anywhere on your filesystem and referenced with
+# the RQD_CONFIG_FILE environment variable or the -c flag when launching rqd.
+# If this file is not found the defaults in rqconstants.py will be used.
+
+[Override]
+RQD_USE_PATH_ENV_VAR=1
+RQD_USE_IP_AS_HOSTNAME=0
+OVERRIDE_IS_DESKTOP=True
+OVERRIDE_NIMBY=True
+# Number of seconds to wait before checking if the user has become idle.
+CHECK_INTERVAL_LOCKED = 60
+# Seconds of idle time required before nimby unlocks.
+MINIMUM_IDLE = 900
+
+[UseHostEnvVar]
```

Comment:
I think a comment line is recommended here to explain what this section is for

Reply:
Good idea, I added some notes.